mubench.reviewsite/tests/create_simple_test_data.php

```diff
@@ -23,6 +23,7 @@
     $table->float('runtime');
     $table->integer('number_of_findings');
     $table->string('result', 30);
+    $table->integer('timestamp');
```

Comment:
I don't like that this script repeats the table creation. Can it use the creation logic from the controllers instead? If you do change this, please ensure it gets into a separate commit!